functorch/_src/aot_autograd.py

```diff
@@ -405,9 +468,15 @@ class CompiledFunction(torch.autograd.Function):
         @staticmethod
         @disable_torchdynamo
         def forward(ctx, *deduped_flat_tensor_args):
-            fw_outs = call_func_with_args(
-                CompiledFunction.compiled_fw, deduped_flat_tensor_args
-            )
+            if disable_amp:
+                with disable_autocast_manager():
```

Comment:
nit: why can't you re-use `context()` here? Since it looks like its already set properly to be either a nullcontext or the disable context.

Reply:
Invoking nullcontext adds a bit of overhead. Since this is on the hot path, I thought better not to use it.

Reply:
agreed it's a little confusing though, will add comment